site/sfguides/src/native-app-chairlift/native-app-chairlift.md

```diff
@@ -696,6 +696,7 @@ DROP DATABASE chairlift_consumer_data;
 
 USE ROLE chairlift_provider;
 DROP APPLICATION PACKAGE chairlift_pkg;
```

Comment:
We should encourage users to use `snow app teardown` instead of dropping the app / pkg in SQL

Reply:
We will address in a subsequent PR.